patches/components-policy-tools-template_writers-template_formatter.py.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/components/policy/tools/template_writers/template_formatter.py b/components/policy/tools/template_writers/template_formatter.py
+index 9af88164b66cb8a718644a9971937b091d2ada45..1ca78c6c83f7dd249686b1ad531fe0421c2ed666 100755
+--- a/components/policy/tools/template_writers/template_formatter.py
++++ b/components/policy/tools/template_writers/template_formatter.py
+@@ -21,7 +21,7 @@ sys.path.insert(
+ 
+ import six
+ 
+-import writer_configuration
++from components.policy.tools.template_writers import brave_writer_configuration as writer_configuration
```

Comment:
Please rewrite this using `inline_file_from_src` approach.
see examples here: https://github.com/brave/brave-core/pull/14693/files

inlined files should be located at the same path in `chromium_src` structure (not `chromium_src/python_modules`!) so we can `touch` them during rebuild the same way we do it for c++ files.

In this file you need to inline our file right before `if '__main__' == __name__:`.

Reply:
Resolved in 4ba1956d4d949f8fa0ebee0c460c5b6a73e96812.